validate_email.py

```diff
@@ -96,6 +97,17 @@ class ServerError(Exception):
 MX_CHECK_CACHE = {}
 
 
+def is_disposable(email, debug=False):
+    """Indicate whether the email is known as being a disposable email or not"""
+    for domain in _disposable:
```

Comment:
no no! do it better
do it like this:

    email_domain = email.rsplit('@',1)[1]
    if email_domain in _disposable:
        # logger stuff
        return True
    return False